### ISSUE TITLE
Crash when trying joining multiplayer server

### ISSUE THREAD

**Issue by blzzxd** [solsticegamestudios/GModCEFCodecFix#136]
So, I installed this GModCEFCodecFix because standart GMod I was unable to write in cyrillic letters. In chat it was just "??? ? ?? ??? ?? ??". Also in console, so I wasn't able to communicate with players. Also x86_64 was crashing for me even in native. When I installed this fix. All worked good, like I was able to write in cyrillics, main menu was loading good. Fonts was not small. I was able to join single player game. BUT when I was trying to connect to the server, all loading fine, but just as I join the server and spawn, I see half frame when I spawn and game instantly crashes...........

What I use:
Steam Runtime
GarrysMod x86_64 branch
Tried with proton and without proton (proton-ge-custom and official ones) With proton my game crash even on loading screen..

**Comment by WinterPhoenix** [comment]
What distro?

**Comment by blzzxd** [comment]
> What distro?

Arch LInux. I found out that this fix doesn't provocating gmod crash when joining the multiplayer server. I played with some settings, and I fixed fonts without this gmod fix, I though thanks god, joined single player mode and I was happy until I tried to join multiplayer server again... It crashes anyway, without GModCEFCodecFix.. So, this fix is not a reason for crash ... I don't know how to fix my gmod anymore. But thanks for reply!

**Comment by WinterPhoenix** [comment]
There's a lot of reasons it could be crashing... If it's happening both with and without Proton, it's likely you have too many addons installed or something.

Try launching GMod with these launch options temporarily:
```
-noaddons -noworkshop
```

**Comment by maxsspeaker** [comment]
-noaddons -noworkshop not fixing that. @WinterPhoenix

**Comment by WinterPhoenix** [comment]
> -noaddons -noworkshop not fixing that. [@WinterPhoenix](https://github.com/WinterPhoenix)

@maxsspeaker Have you read the FAQ?

**Comment by maxsspeaker** [comment]
> > -noaddons -noworkshop not fixing that. [@WinterPhoenix](https://github.com/WinterPhoenix)
> 
> [@maxsspeaker](https://github.com/maxsspeaker) Have you read the FAQ?

yes, I read the FAQ... This crash occurs on certain servers. Switching to Proton Experimental fixes the crash. But it's strange.

**Comment by WinterPhoenix** [comment]
Maybe you're just running out of memory or something?

**Comment by maxsspeaker** [comment]
> Maybe you're just running out of memory or something?

The game crashes immediately after loading is complete—only half of the 32 GB of RAM is used...

If you play the Windows version, though, it takes up almost all 32 GB (some of it goes to swap space). But it doesn't crash. 
I can provide the IP address of this server for testing.
